package.json

```diff
@@ -60,10 +60,10 @@
   "dependencies": {
     "babel-plugin-debug-macros": "^0.3.3",
     "ember-cli-babel": "^7.13.0",
-    "ember-compatibility-helpers": "^1.2.0-beta.1"
-  },
-  "peerDependencies": {
-    "ember-data": ">= 3.5.1"
+    "@ember-data/store": "^3.16.0-alpha.2",
```

Comment:
If users have different versions of ember-data, strange duplication will occur. A solution to that is a pre-req to this approach.

Reply:
This PR contains a solution to this already. If the app already contains EmberData, we exclude our own versions of the addons.

Reply:
Can you share a link to the code that implements that?

Reply:
https://github.com/hjdivad/ember-m3/pull/478/files#diff-168726dbe96b3ce427e7fedce31bb0bcR24-R37

Reply:
The detection is a bit more raw than our other detection logic in the other hooks because ember-cli calls this hook prior to setting up addons, if you can think of ways to improve it I'm all 👂

Reply:
that link doesn't appear to work for me..